alerta/utils/config.py

```diff
@@ -46,77 +137,97 @@ def get_user_config():
         # Use app config for DATABASE_URL if no env var from above override it
         config['DATABASE_URL'] = database_url or config['DATABASE_URL']
 
+        if config['DATABASE_URL'] in os.environ:
+            config['DATABASE_URL'] = valid.validate_url(os.environ['DATABASE_URL'])
+
         if 'DATABASE_NAME' in os.environ:
-            config['DATABASE_NAME'] = os.environ['DATABASE_NAME']
+            config['DATABASE_NAME'] = valid.validate_string(os.environ['DATABASE_NAME'])
 
         if 'AUTH_REQUIRED' in os.environ:
-            config['AUTH_REQUIRED'] = True if os.environ['AUTH_REQUIRED'] == 'True' else False
+            config['AUTH_REQUIRED'] = valid.validate_boolean(os.environ['AUTH_REQUIRED'])
 
         if 'AUTH_PROVIDER' in os.environ:
-            config['AUTH_PROVIDER'] = os.environ['AUTH_PROVIDER']
+            config['AUTH_PROVIDER'] = valid.validate_string(os.environ['AUTH_PROVIDER'])
 
         if 'ADMIN_USERS' in os.environ:
-            config['ADMIN_USERS'] = os.environ['ADMIN_USERS'].split(',')
+            list = valid.validate_list(os.environ['ADMIN_USERS'])
+            config['ADMIN_USERS'] = valid.validate_string(list)
```

Comment:
Why do you validate a list and then pass that list into a method that validates the list as a string? It doesn't make much sense.

It would be more consistent to do something similar to the other variable types, wouldn't it ...
```
config['ADMIN_USERS'] = valid.validate_list(os.environ['ADMIN_USERS'].split(','))
```
And do all the validation for whatever type in the validation methods. But maybe I'm missing something.

Reply:
The goal was to first parse and validate the input as a list and then validate that all members of that list was string. 
Maybe change the name of method `validate_list` to `parse_list`. 
And then make a `validate_string_list` method to validate that all variables in the list is strings. 

I found out that voluptuous has a schema ready for checking if all members of a list is string: 
`list_string_validator = Schema([str])`

If all members of a list should always be treated as strings I can make it more similar to the others I can combine the pars_list and validate_list into one method?